src/helpers/apikey.ts

```diff
@@ -1,28 +1,57 @@
-import axios from 'axios'
+import {AxiosError, default as axios} from 'axios'
+import chalk from 'chalk'
+import {BufferedMetricsLogger} from 'datadog-metrics'
 
+import {InvalidConfigurationError} from './errors'
+import {} from './utils'
```

Comment:
I'm not sure if our linter allows this, but you should be able to write this as `import './utils'`.
Also, why is it needed to import the file but no function from it?

Reply:
Strange, that must be a leftover. I removed it!